landoapi/revisions.py

```diff
@@ -118,3 +119,26 @@ def check_author_planned_changes(*, revision, **kwargs):
         return None
 
     return "The author has indicated they are planning changes to this revision."
+
+
+def revision_is_secure(revision, phabclient):
+    """Does the given revision contain security-sensitive data?
+
+    Such revisions should be handled according to the Security Bug Approval Process.
+    See https://wiki.mozilla.org/Security/Bug_Approval_Process.
+
+    Args:
+        revision: The 'data' element from a Phabricator API response containing
+            revision data.
```

Comment:
This isn't completely correct, it specifically needs to be a `differential.revision.search` response for a revision, which included the `projects` attachment. We have a convention for pulling all the data for the stack in the one place, so that it has all we need and is standard, you could refer to the stack data stuff if you want this comment to require less updating.

Reply:
I used the docstring from another function with a "revision" data argument as a template to clarify this docstring.